docs/faq-user.md

```diff
@@ -20,37 +20,32 @@ While there is a delay before you can access your previously delegated tokens, t
+Currently, slashing is not enabled on the Casper Mainnet. If a validator behaves poorly on the network, they may be evicted from the auction and you will not earn rewards during the period that the validator is evicted. When slashing is enabled, tokens will be removed as penalty for poor/malicious behavior in the network. In that case, you will lose tokens delegated to that validator.
 
 ## What is the cost of delegation and undelegation?
 
-The cost of the delegation process is ~3 CSPR. So make sure you have a bunch of extra CSPR on your account apart from the amount you are delegating. So if you are going to delegate 1000 CSPR, you should have at least 1003 CSPR in your wallet.The cost of the undelegate process is currently ~0.5 CSPR.
+The cost of the delegation process is 2.5 CSPR. So make sure you have a bunch of extra CSPR on your account apart from the amount you are delegating. So if you are going to delegate 1000 CSPR, you should have at least 1002.5 CSPR in your wallet. The cost of the undelegate process is currently also 2.5 CSPR.
 
 ## What is the yearly reward rate for the delegated tokens?
 
-The base annual reward rate is 8% of the total supply, resulting in a current APY of approximately 20% on the MainNet (because only a portion of the total supply is currently staked and participating in the reward distribution). The APY is likely to decrease over time as more tokens get staked on the network.
+The base annual reward rate is 8% of the total supply, resulting in a current APY of approximately 15% on the Mainnet (because only a portion of the total supply is currently staked and participating in the reward distribution). The APY might decrease over time as more tokens get staked on the network.
 
 ## I see some validators with big stake producing blocks, and some smaller ones not. Does that mean only the bigger ones are getting rewards?
 
-No. On Casper Mainnet, everyone gets seigniorage according to their stake, regardless of whether you've produced blocks or not. The only thing a block proposer gets, is to keep the transaction fees associated with transfers and deploys in the block they proposed. As you may know, transfers cost only 10000 motes (0.00001 CSPR), so these fees are not substantial on a transfer-by-transfer basis.
+No. On Casper Mainnet, everyone gets seigniorage according to their stake, regardless of whether you've produced blocks or not. The only thing a block proposer gets, is to keep the transaction fees associated with transfers and deploys in the block they proposed. As you may know, transfers cost only 100000000 motes (0.1 CSPR), so these fees are not substantial on a transfer-by-transfer basis.
```

Comment:
@mrkara this I don't think is accurate anymore? (that the proposer keeps the fees). Please verify and adjust if needed.

Reply:
@mssteuer Just checked. It's still the case.